<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Casino Game</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../dollar-sign.svg</normaloff>../dollar-sign.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #4F46E5;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="betBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monospace</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	padding: 16px;
	font-size: 20px;
	border-radius: 8px;
	background-color: #10B981;
	color: #fff;
	outline: none;
	font-family: monospace;
}

QPushButton:hover {
	background-color: #34D399;
}

QPushButton:focus {
	outline: none;
}
</string>
    </property>
    <property name="text">
     <string>Bet</string>
    </property>
   </widget>
   <widget class="QLabel" name="depositMoneyLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F3F4F6;
font-size: 32px;</string>
    </property>
    <property name="text">
     <string>Deposit Money: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="betInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border-radius: 10px;
	background-color: #fff;
	color: #000;
	padding: 13px;
	font-size: 20px;
	text-align: center;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentBetLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color: #fff;</string>
    </property>
    <property name="text">
     <string>Current Bet:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="guessInput">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border-radius: 10px;
	background-color: #fff;
	color: #000;
	padding: 13px;
	font-size: 20px;
	text-align: center;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountBetLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color: #fff;</string>
    </property>
    <property name="text">
     <string>Amount to bet</string>
    </property>
   </widget>
   <widget class="QLabel" name="guessLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color: #fff;</string>
    </property>
    <property name="text">
     <string>Your guess</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctNumberLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color: #fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="guessLabelDisplay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color: #fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
